src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -127,6 +127,10 @@ module sonic-device_metadata {
                     type string;
                 }
 
+                leaf sub_role {
+                    type string;
```

Comment:
Please add description.

Reply:
Hi @venkatmahalingam  Could you provide a description message?

Reply:
Could use the description provided here: https://github.com/Azure/SONiC/blob/master/doc/multi_asic/SONiC_multi_asic_hld.md#2432-device_metadata